src/server/pkg/storage/fileset/storage.go

```diff
@@ -12,6 +12,8 @@ import (
 const (
 	prefix    = "pfs"
 	headerTag = ""
+	// (bryce) explain this somewhere.
```

Comment:
I added an explanation, but I am still figuring out how I should be setting up the whole representation of full vs symbolic merges. I am thinking the model I mentioned in the symbolic merge section of https://github.com/pachyderm/pachyderm/issues/3704 could work nicely, but it would require us to commit (pun intended) to the model of incrementing commit identifiers.